include/invert_quda.h

```diff
@@ -427,7 +430,10 @@ namespace quda {
     CG(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile);
     virtual ~CG();
 
-    void operator()(ColorSpinorField &out, ColorSpinorField &in);
+    void operator()(ColorSpinorField &out, ColorSpinorField &in){
```

Comment:
Doxygen markup needed here for these `operator()` member functions.

Also I'd suggest we rename `pin` to `p_in` (I first read this as "pin" and was trying to workout if the field was somehow "pinned").